modules/ref-dashboard-configuration-options.adoc

```diff
@@ -83,63 +68,44 @@ On the *Deploy model* dialog when using the single-model serving platform:
-| `spec.dashboardConfig.
-disableStorageClasses` | `false` | Shows the *Settings → Storage classes* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
-| `spec.dashboardConfig.
-disableSupport` | `false` | Shows the *Support* menu item when a user clicks the Help icon in the dashboard toolbar. To hide this menu item, set the value to `true`.
+| `spec.dashboardConfig.disableModelServing` | `false` | Shows the *Models → Model deployments* menu item in the dashboard navigation menu, and the *Models* tab in data science projects. To hide these items, set the value to `true`. 
+| `spec.dashboardConfig.disableNIMModelServing` | `false` | Enables the ability to select NVIDIA NIM as a model-serving platform. To disable this ability, set the value to `true`.
+| `spec.dashboardConfig.disablePerformanceMetrics` | `false` | Shows the *Endpoint Performance* tab on the *Model deployments* page. To hide this tab, set the value to `true`.
+| `spec.dashboardConfig.disablePipelines` | `false` | Shows the *Data science pipelines* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
+| `spec.dashboardConfig.disableProjects` | `false` | Shows the *Data science projects* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
+| `spec.dashboardConfig.disableProjectSharing` | `false` | Allows users to share access to their data science projects with other users. To prevent users from sharing data science projects, set the value to `true`.
+| `spec.dashboardConfig.disableServingRuntimeParams` | `false` | Shows the *Configuration parameters* section in the *Deploy model* dialog and the *Edit model* dialog when using the single-model serving platform. To hide this section, set the value to `true`.
+| `spec.dashboardConfig.disableStorageClasses` | `false` | Shows the *Settings → Storage classes* menu item in the dashboard navigation menu. To hide this menu item, set the value to `true`.
+| `spec.dashboardConfig.disableSupport` | `false` | Shows the *Support* menu item when a user clicks the Help icon in the dashboard toolbar. To hide this menu item, set the value to `true`.
 ifdef::upstream[]
-| `spec.dashboardConfig.
-disableTracking` | `true` | Disables the collection of data about {productname-short} usage in your cluster. To enable data collection, set the value to `false`. You can also set this option in the {productname-short} dashboard interface from the *Settings → Cluster settings* navigation menu.	
+| `spec.dashboardConfig.disableTracking` | `true` | Disables the collection of data about {productname-short} usage in your cluster. To enable data collection, set the value to `false`. You can also set this option in the {productname-short} dashboard interface from the *Settings → Cluster settings* navigation menu.	
```

Comment:
is this a duplicate? I see the same in line 101

Reply:
It's conditionalised text because that config option has a different value upstream and downstream.